modules/cloudsql-instance/variables.tf

```diff
@@ -90,6 +90,16 @@ variable "deletion_protection_enabled" {
   nullable    = false
 }
 
+variable "deny_maintenance_period" {
```

Comment:
Can we collapse this and `maintenance_window` into a single `maintenance_config` variable? We always try really hard to keep our variable space small.

Reply:
Done! Thank you